app/models/league/match.rb

```diff
@@ -219,18 +219,18 @@ def validate_home_and_away_team_are_in_the_same_division
       return unless home_team.present? && away_team.present?
 
       unless away_team.division == home_team.division
-        errors.add(:away_team, 'must be in the same division as the home team')
+        errors.add(:away_team, 'Must be in the same division as the home team')
```

Comment:
Pretty sure the text generated from this is: 'Away team must be in the same division as the home team', hence the lower case.

Reply:
Noted, correcting.

Reply:
Welp, I was wrong. My rails is getting a little rusty :)